README.md

```diff
@@ -22,6 +22,88 @@ Java objects.
 
 ## Installation
 
-## Example
+Add this library as a dependency to your project's `pom.xml` file.
+
+```xml
+<dependencies>
+    <dependency>
+        <groupId>com.exasol</groupId>
+        <artifactId>parquetio</artifactId>
+        <version>LATEST VERSION</version>
+    </dependency>
+</dependencies>
+```
+
+Please use the latest version of the library.
+
+## Usage
+
+Here is a small example code showing the usage of the library.
+
+```java
+final Path path = new Path("/data/parquet/part-0000.parquet");
+final Configuration conf = new Configuration();
+try (final ParquetReader<Row> reader = RowParquetReader
```

Comment:
No, unfortunately it is not possible.
It accepts `InputFile` interface, but only implementation is `HadoopInputFile`. Additionally, it requires `Configuration` object, it will create it itself underneath if not provided.

Reply:
then I would suggest mentioning the required libs in the README.md